.github/workflows/scripts/generate-playground-blueprint.js

```diff
@@ -0,0 +1,92 @@
+const https = require( 'https' );
+
+const generateWordpressPlaygroundBlueprint = ( runId, prNumber ) => {
+	const defaultSchema = {
+		landingPage: '/wp-admin/admin.php?page=wc-admin',
+
+		preferredVersions: {
+			php: '8.0',
+			wp: 'latest',
+		},
+
+		phpExtensionBundles: [ 'kitchen-sink' ],
+
+		features: { networking: true },
+
+		steps: [
+			{
+				step: 'installPlugin',
+				pluginZipFile: {
+					resource: 'url',
+					url: `https://playground.wordpress.net/plugin-proxy.php?org=woocommerce&repo=woocommerce&workflow=Build%20Live%20Branch&artifact=plugins-${ runId }&pr=${ prNumber }`,
+				},
+				options: {
+					activate: true,
+				},
+			},
+			{
+				step: 'login',
+				username: 'admin',
+				password: 'password',
+			},
+		],
+		plugins: [],
```

Comment:
Is it possible to add the Beta tester here?

Reply:
Oh, I see the comment now that we can't. Thanks this is fantastic!

Reply:
@psealock I don't think it's something I can influence or work on right now, but I think we should set up like a basic AWS s3 bucket or something to serve up a few of these plugins like smooth generator with cors headers.